app/v1/models/party.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Party model.
+
+This module helps data manipulation.
+PartyOffice class has multiple methods that manipulate,
+the party data received from party views.
+"""
+
+# Mock data
+DB = [
+    {
+        "id": 1,
+        "name": 'party A',
+        "hqAddress": 'box 432, Nairobi',
+        "logoUrl": 'https://via.placeholder.com/150'
+    },
+    {
+        "id": 2,
+        "name": 'party B',
+        "hqAddress": 'box 148, Nairobi',
+        "logoUrl": 'https://via.placeholder.com/150'
+    }
+]
+
+
+class PartyModel:
+    """Office Model class"""
+
+    def __init__(self):
+        self.db = DB
+
+    def find_party(self, pid: int) -> dict:
+        """
+        Finds a specific party
+
+        Args:
+        pid: party Id.
+
+        Returns:
+            Returns patry dictionary for party found
+            Rerurns empty list for party not found.
+        """
+
+        party_found = [party for party in self.db if party['id'] == pid]
+        return party_found
+
+    def delete_party(self, pid: int):
+        """
+        Delete party from list
+
+        Args:
+        oid: party id.
```

Comment:
The name of the argument for the function is **pid** while in this documentation you are using **oid**, correct this.